web-app/src/app/store/saga/auth-saga.ts

```diff
@@ -80,10 +79,9 @@ function* logoutSaga({
   navigateTo: NavigateFunction;
 }>): Generator {
   try {
+    navigateTo(redirectScreen);
```

Comment:
We are navigating out of the current screen before logging out to avoid race conditions.